src/app/atoms/FormSecondaryButton.tsx

```diff
@@ -22,37 +22,32 @@ export const FormSecondaryButton: FC<FormSecondaryButtonProps> = ({
   style,
   children,
   ...rest
-}) => (
-  <Button
-    type={type}
-    className={classNames(
-      'relative',
-      small ? 'px-6' : 'px-8',
-      'bg-white rounded',
-      'border-2 border-primary-orange',
-      'flex items-center',
-      loading ? 'text-transparent' : 'text-primary-orange',
-      small ? 'text-sm' : 'text-base',
-      'font-semibold',
-      'transition duration-200 ease-in-out',
-      loading || disabled ? 'opacity-75' : 'opacity-90 hover:opacity-100',
-      loading || disabled ? 'shadow-inner pointer-events-none' : 'shadow-sm hover:shadow focus:shadow',
-      className
-    )}
-    style={{
-      paddingTop: small ? '0.5rem' : '0.625rem',
-      paddingBottom: small ? '0.5rem' : '0.625rem',
-      ...style
-    }}
-    disabled={disabled}
-    {...rest}
-  >
-    {children}
+}) => {
+  const classNameMemo = useMemo(
```

Comment:
Why do we use `className` in the singular but not `classNames? And in the other cases